packages/localize-monetary-amount/test/localize-monetary-amount.js

```diff
@@ -0,0 +1,358 @@
+/**
+ * Internal dependencies
+ */
+import { localizeMonetaryAmount } from '../src/index';
+
+describe( 'localizeMonetaryAmount', function() {
+	describe( 'locale is en-us', function() {
+		it( 'en-us/USD, 0', function() {
+			expect( localizeMonetaryAmount( 'en-us', 'USD', 0 ) ).toBe( '$0' );
+		} );
+		it( 'en-us/USD, 5', function() {
+			expect( localizeMonetaryAmount( 'en-us', 'USD', 5 ) ).toBe( '$0.05' );
+		} );
+		it( 'en-us/USD, 50', function() {
+			expect( localizeMonetaryAmount( 'en-us', 'USD', 50 ) ).toBe( '$0.50' );
+		} );
```

Comment:
This doesn't need to be rewritten, but these very repetitive tests can be written concisely and expressively with [```test.each`table`(…)```](https://jestjs.io/docs/en/api#2-testeach-table-name-fn-timeout).

Reply:
TIL!